README.md

```diff
@@ -137,27 +132,20 @@ Install-packageProvider -Name NuGet -Force
 Install-Module -Name AzureRM -Force
 ```
 
- 1. Run Microsoft Powershell.
- 1. Create the local parameters file by calling
-   ```
-   Invoke-Webrequest -Uri "https://raw.githubusercontent.com/teradici/deploy/master/azuredeploy.parameters.json" -OutFile "my.azuredeploy.parameters.json"
-   ```
- 3. Modify <samp>my.azuredeploy.parameters.json</samp> to include the necessary deployment parameters.
- 1. Run the following script, substituting username, password, resource group name, and desired region:
-
-```
-$spUsername = "<username>@<example>.com"
-$spPass = ConvertTo-SecureString "<password>" -AsPlainText -Force
-$cred = New-Object -TypeName pscredential -ArgumentList $spUsername, $spPass
-Login-AzureRMAccount -Credential $cred
-
-$azureRGName = "<rgname>"
-New-AzureRMResourceGroup -Name $azureRGName -Location "East US"
-New-AzureRMResourceGroupDeployment -DeploymentName "ad1" -ResourceGroupName $azureRGName -TemplateFile "https://raw.githubusercontent.com/teradici/deploy/master/azuredeploy.json" -TemplateParameterFile "my.azuredeploy.parameters.json"
-
-```
-
-If you do not want credentials in the file just go directly to <samp>Login-AzureAccount</samp> without the <samp>-Credential</samp> parameter and it will give you a prompt.
+1. Run Microsoft Powershell.
```

Comment:
Are you trying to convey how to run in a local PowerShell environment instead of Cloud Shell?

Reply:
Yes, just updating the previous information about how to run it locally from powershell to the new ones.

Reply:
@bill-dall do you want this section on deploying Cloud Access Manager with PowerShell to be added to the admin guide?

Reply:
Please see the "CAM deployment doc's" email I sent a few days ago to see what I've got an opinion on or not. I think you and Charanpreet were supposed to arm-wrestle on some things.